en/technical-reference/api-header.md

```diff
@@ -71,11 +85,9 @@ curl -X POST \
 
 #### Exception on Frontend API Request
 
-For API request that is intended to be called from frontend/client side, such as API `/token` (get card token), etc. Above usual headers **are not required**, and you **should not use Server Key** to authorize the request, to avoid the risk of exposing your Server Key on your publicly accessible frontend.
+For API request from your side, such as GET Card Token API, the headers explained earlier **are not required**. To avoid the risk of exposing your *Server Key* on your publicly accessible frontend, you **should not** use *Server Key* to authorize the request. Instead, *Client Key* is used to authorize the HTTP(s) request.
```

Comment:
`frontend/client side` is a technical terms. Please do not replace it with `your side`